sensor_msgs/CompressedImage annotated_img
string[] classes
float64[] bboxes
sensor_msgs/CompressedImage[] masks